app/controllers/kudos_controller.rb

```diff
@@ -46,6 +47,10 @@ def destroy
     redirect_to kudos_url, notice: 'Kudo was successfully destroyed.'
   end
 
+  def correct_employee
+    redirect_to kudos_path, notice: "You aren't authorized to edit this Kudo." unless @kudo.giver_of_kudo == current_employee
```

Comment:
I would change _edit_ to _access_ in the notice, since you are not only editing it, but also destroying.

```suggestion
    redirect_to kudos_path, notice: "You aren't authorized to access this Kudo." unless @kudo.giver_of_kudo == current_employee
```

Reply:
Done!